src/App.js

```diff
@@ -1,12 +1,184 @@
 import React from 'react';
 import './App.css';
+import {useForm} from 'react-hook-form';
+import FruitCounter from "./components/FruitCounter";
+import LabelAndInput from "./components/LabelAndInput";
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    // Destructuring register and handleSubmit functions from useForm
+    const {register, handleSubmit} = useForm();
+
+    // Handle data from fruitcounter and order form
+    function onFormSubmit(data) {
+        console.log(`
+        Persoonsgegevens:\nVoornaam: ${data['first-name']}\nAchternaam: ${data['last-name']}\nLeeftijd: ${data.age}\nPostcode: ${data.zipcode}\nBezorgfrequentie: ${data['deliver-frequency']}\nBezorgmoment: ${data['deliver-moment']}\nOpmerking: ${data.comments}\nAkkoord met de voorwaarden: ${data['terms-and-conditions']}
+        `
+        );
+        console.log(`
+            Bestelling:\n${count.strawberry} aardbeien\n${count.banana} bananen\n${count.banana} appels\n${count.kiwi} kiwi's
+        `
+        );
+    }
+
+    // Set state for multiple inputs
+    const [count, setCount] = React.useState({
+        strawberry: 0,
+        banana: 0,
+        apple: 0,
+        kiwi: 0,
+    })
```

Comment:
Ik zou voor elk fruitsoort een aparte useState gebruiken. Het werkt goed zo hoor, maar nu zal elke update van de `count` een onnodige rerender veroorzaken voor het fruit wat niet verlaagd of opgehoogd wordt